DEV_GUIDE.md

```diff
@@ -133,3 +139,19 @@ curl -L -X GET 'localhost:8082/v1/keyspaces' \
 --header 'content-type: application/json' \
 --header 'X-Cassandra-Token: <AUTH_TOKEN>'
 ```
+
+## Running Tests
+
+Stargate includes a full suite of integration tests that can be ran in various ways. Some examples are
```

Comment:
```suggestion
Stargate includes a full suite of integration tests that can be ran in various ways. Some examples are:
```

Reply:
Thanks! Might have you look through the generated swagger UI later to help clean up.